server/main.go

```diff
@@ -104,15 +104,16 @@ func (c *Server) handle(conn *net.TCPConn) error {
 	switch request.Type {
 	case btd.ISSUE:
 		metrics.CounterIssueTotal.Inc()
-		err = btd.HandleIssue(conn, request, c.key, c.G, c.H, c.MaxTokens)
+		// use the first key as issue key
```

Comment:
also, if we only support loading the first key from the file this will have to be documented in the future. Maybe if we had multiple files we could support loading a default signing key and then extra optional key files for redemption from the command line. This might be a bit clearer, what do you think?